src/main/java/com/jfrog/ide/idea/ui/LocalComponentsTree.java

```diff
@@ -108,24 +109,28 @@ private void handleContextMenu(ComponentsTree tree, MouseEvent e) {
             return;
         }
         // Event is right-click.
+
         TreePath selectedPath = tree.getPathForRow(tree.getClosestRowForLocation(e.getX(), e.getY()));
         if (selectedPath == null) {
             return;
         }
         Object selected = selectedPath.getLastPathComponent();
+
         if (selected instanceof DependencyNode) {
-            createNodePopupMenu((DependencyNode) selected);
+            DescriptorFileTreeNode descriptorFileTreeNode = (DescriptorFileTreeNode) selectedPath.getParentPath().getLastPathComponent();
+            String descriptorPath =  descriptorFileTreeNode.getSubtitle();
+            this.createNodePopupMenu((DependencyNode) selected, descriptorPath);
         } else if (selected instanceof VulnerabilityNode) {
             createIgnoreRuleOption((VulnerabilityNode) selected, e);
         } else if (selected instanceof ApplicableIssueNode) {
             createIgnoreRuleOption(((ApplicableIssueNode) selected).getIssue(), e);
         } else {
```

Comment:
instead of removing the comment here - I suggest adding comments to all for cases of the if/else

Reply:
okay tell me your opinion